sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -739,6 +754,16 @@ public Write withEmulator(String emulatorHost) {
       return toBuilder().setBigtableConfig(config.withEmulator(emulatorHost)).build();
     }
 
+    /* This is an experimental feature that may get changed in the future
+     *
+     * Returns a new {@link BigtableIO.Write} that will report amount of time throttling to Dataflow
+     */
+    @Experimental
+    public Write withDataflowThrottleReporting() {
```

Comment:
I would add a bool param here. It will allow you to transition this to be opt out later